src/Microsoft.DotNet.Build.Tasks/GenerateTestExecutionScripts.cs

```diff
@@ -109,29 +112,29 @@ private void WriteCmdExecutionScript(string cmdExecutionTemplate, string outputP
             StringBuilder copyCommands = new StringBuilder();
             foreach (ITaskItem dependency in TestDependencies)
             {
-                string relativePath = dependency.GetMetadata("PackageRelativePath");
+                string relativeDestinationPath = dependency.GetMetadata("RelativeDestinationPath").Replace('/', '\\');
```

Comment:
This code and the code for WriteShExecutionScript appear to be very similar.  Could this be refactored to eliminate duplication and so that making an update in one place updates both scripts?  Perhaps a method with an optional char separator of '\\' or '/'